app/Services/User/LogoutUserService.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Services\User;
+
+use App\Http\Resources\ErrorResource;
+use App\Http\Resources\SuccessResource;
+use App\Repositories\UserRepository;
+use Exception;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+
+class LogoutUserService
+{
+    protected $userRepository;
+
+    public function __construct(UserRepository $userRepository)
+    {
+        $this->userRepository = $userRepository;
+    }
+
+    public function execute(Request $request)
+    {
+        try {
+            $request->user()->currentAccessToken()->delete();
+            return new SuccessResource(Response::HTTP_CREATED, "Logged out successfully.");
```

Comment:
This is HTTP_OK status code. We use HTTP_CREATED just when we successfully stored a record in DB.

Reply:
resolved